test/functional/p2p_invalid_tx.py

```diff
@@ -54,8 +55,7 @@ def new_spend_tx(self, prev_hash, prev_n, values):
         anyone-can-spend script"""
         tx = CTransaction()
         tx.vin.append(CTxIn(outpoint=COutPoint(prev_hash, prev_n)))
-        for value in values:
-            tx.vout.append(CTxOut(nValue=value, scriptPubKey=CScript([OP_TRUE])))
+        for value in values: tx.vout.append(value)
```

Comment:
nit: the variable name `values` is confusing now. How about changing it to e.g. `tx_outs`, and then doing simply `tx.vout = tx_outs` ?

Reply:
yeah done.